style.css

```diff
@@ -464,124 +433,120 @@ ul li {
   display: flex;
   align-items: center;
   color: #253858;
-  width: 303px;
+  width: 100%;
   height: 66.72px;
   background: #f7f7f9;
   border-radius: 8px;
 }
 
 #framework {
-  top: 828px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
 }
 
 .arrow1 {
-  position: absolute;
-  border: 2px solid #fff;
-  top: 830px;
-  left: 75.5%;
-}
-
-#skills {
-  top: 900px;
-}
-
-.arrow2 {
-  position: absolute;
-  border: 2px solid #fff;
-  top: 900px;
-  left: 75.5%;
+  left: 41.67%;
+  right: 33.33%;
+  top: 25%;
+  bottom: 25%;
+  border: 2px solid #acb7c3;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  border-radius: 8px;
 }
 
 .separator1 {
-  position: absolute;
   height: 1px;
-  left: 24px;
+  left: 0;
   right: 0;
   bottom: 0;
-  width: 327px;
-  top: 875px;
-  border-radius: 0;
   background: #dfe1e6;
 }
 
+#skills {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+.arrow2 {
+  left: 41.67%;
+  right: 33.33%;
+  top: 25%;
+  bottom: 25%;
+  border: 2px solid #acb7c3;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  border-radius: 8px;
+}
+
 .separator2 {
-  position: absolute;
   height: 1px;
-  left: 24px;
+  left: 0;
   right: 0;
   bottom: 0;
-  width: 327px;
-  top: 947px;
-  border-radius: 0;
   background: #dfe1e6;
 }
 
 .contactme {
   background: url(img/cf_bg.svg), url(img/cf_bf.svg), #6070ff;
   background-position: 100% 100%;
+  background-size: contain;
   background-repeat: no-repeat;
-  position: absolute;
-  width: 375px;
-  height: 779px;
-  left: 0;
-  top: 4680px;
-  border-radius: 20% 0 0 0/10% 0 0 0;
+  border-top-left-radius: 70px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 24;
 }
 
 .contacttxt {
-  position: absolute;
-  width: 375px;
-  height: 56px;
-  left: 0;
-  top: 86px;
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
   display: flex;
-  justify-content: center;
+  align-items: center;
+  text-align: center;
   color: #fff;
 }
 
 .paragraph {
-  position: absolute;
-  width: 327px;
-  height: 102px;
-  left: 24px;
-  top: 170px;
   font-style: normal;
   font-weight: 400;
   font-size: 20px;
   line-height: 28px;
   text-align: center;
+  margin-top: 12px;
   color: #ebebff;
 }
 
-.radius {
-  border-radius: 8px;
-}
-
-.inputtxt1 {
-  position: relative;
-  left: 24px;
-  right: 0;
-  top: 368px;
-  bottom: 0;
-  height: 48px;
-  width: 327px;
-  border-radius: 8px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
+#form-id {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 52px;
+  gap: 1rem;
 }
 
+.inputtxt1,
 .inputtxt2 {
```

Comment:
- [ ] kindly as requested by the previous reviewer consider checking on this 👉  In your design, you used the wrong color for your all your placeholder text, which is different from the Figma design given. please ensure you use the appropriate colour to give you a good aesthetic that'd attract hiring managers 😄